src/helpers/utils.ts

```diff
@@ -72,5 +72,5 @@ export function getIp(req) {
     ''
   ).split(',');
 
-  return ips[0].trim();
+  return sha256(ips[0].trim()).substring(0, 10);
```

Comment:
You should hash after retrieving that value. Editing the returned value is breaking current implementation

Reply:
Why 🤔 where are only using this as a key right 🤔

Reply:
Which one is breaking? how can i reproduce?

Reply:
Test are breaking. I think shutter is also using the unhashed IP

Reply:
Good point, let's keep the function getIp unchanged and hash later in rateLimit file

Reply:
We should do the same on score-api and hub